<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindWalkerEarthingMainDlg</class>
 <widget class="QDialog" name="WindWalkerEarthingMainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wind Walker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="windwalker.qrc">
    <normaloff>:/Other/images/Other/windwalker (2).png</normaloff>:/Other/images/Other/windwalker (2).png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请为您的工程取一个适当的名字，虽然程序允许但是不建议使用重复的名字&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>管理您的工程：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="btn_ADD">
         <property name="toolTip">
          <string>添加一个工程</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="windwalker.qrc">
           <normaloff>:/GroundingCAD/images/GroundingCAD/add.png</normaloff>:/GroundingCAD/images/GroundingCAD/add.png</iconset>
         </property>
         <property name="shortcut">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_Del">
         <property name="toolTip">
          <string>删除所选工程</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="windwalker.qrc">
           <normaloff>:/GroundingCAD/images/GroundingCAD/delete.png</normaloff>:/GroundingCAD/images/GroundingCAD/delete.png</iconset>
         </property>
         <property name="shortcut">
          <string>Del</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_Up">
         <property name="toolTip">
          <string>上移所选工程</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="windwalker.qrc">
           <normaloff>:/GroundingCAD/images/GroundingCAD/arrow_up.png</normaloff>:/GroundingCAD/images/GroundingCAD/arrow_up.png</iconset>
         </property>
         <property name="shortcut">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_down">
         <property name="toolTip">
          <string>下移所选工程</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="windwalker.qrc">
           <normaloff>:/GroundingCAD/images/GroundingCAD/arrow_down.png</normaloff>:/GroundingCAD/images/GroundingCAD/arrow_down.png</iconset>
         </property>
         <property name="shortcut">
          <string>Down</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="btn_Input">
          <property name="toolTip">
           <string>请输入相应接地网的参数</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="windwalker.qrc">
            <normaloff>:/Other/images/Other/输入.png</normaloff>:/Other/images/Other/输入.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Compute">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>开始接地分析数据，需要正确输入参数后才能开启</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="windwalker.qrc">
            <normaloff>:/Other/images/Other/计算.png</normaloff>:/Other/images/Other/计算.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Output">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>查看计算结果</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="windwalker.qrc">
            <normaloff>:/Other/images/Other/输出.png</normaloff>:/Other/images/Other/输出.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>参数输入</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QToolButton" name="toolBtn_GroundCAD">
                    <property name="toolTip">
                     <string>点击输入接地网参数</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="windwalker.qrc">
                      <normaloff>:/Other/images/Other/grid.png</normaloff>:/Other/images/Other/grid.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>48</width>
                      <height>48</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>接地网结构</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>14</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QToolButton" name="toolBtn_Earth">
                    <property name="toolTip">
                     <string>点击输入土壤分层结构</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="windwalker.qrc">
                      <normaloff>:/Other/images/Other/earth.png</normaloff>:/Other/images/Other/earth.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>48</width>
                      <height>48</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>土壤结构</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>输出图形</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>10</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolBtn_Plot3D">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="windwalker.qrc">
                    <normaloff>:/Other/images/Other/卡通动物图标下载6.png</normaloff>:/Other/images/Other/卡通动物图标下载6.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>观测面</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="windwalker.qrc">
                    <normaloff>:/Other/images/Other/text.png</normaloff>:/Other/images/Other/text.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>观测线</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>10</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="windwalker.qrc"/>
 </resources>
 <connections/>
</ui>
